docker/internal-auth/env.template

```diff
@@ -0,0 +1,65 @@
+# This should be the URL your end-users will connect to
+BASE_URL=https://example.fiftyone.ai
+
+# This should be set to the URI your end-users will use to connect to the API
+#  This could be the same as AUTH0_BASE_URL
+FIFTYONE_API_URI=https://example-api.fiftyone.ai
+
+# This should be a MongoDB Connection String for your database
+FIFTYONE_DATABASE_URI="mongodb://username:password@mongodb-example.fiftyone.ai:27017/?authSource=admin"
+# If you are using a different MongoDB Connection String for your CAS database,
+#  set it here
+# CAS_DATABASE_URI=""mongodb://username:password@mongodb-cas-example.fiftyone.ai:27017/?authSource=admin""
+
+# FIFTYONE_AUTH_SECRET is a random string used to authenticate to the CAS service
+FIFTYONE_AUTH_SECRET=
+
+# This key is required and is used to encrypt storage credentials in the MongoDB
+#   do NOT lose this key!
+# generate keys by executing the following in python:
+#
+# from cryptography.fernet import Fernet
+# print(Fernet.generate_key().decode())
+#
+FIFTYONE_ENCRYPTION_KEY=
+
+# FiftyOne Teams API container configuration
+API_BIND_ADDRESS=127.0.0.1
+API_BIND_PORT=8000
+API_LOGGING_LEVEL=INFO
+# The following is a docker-compose link and will work in most situations
+API_URL=http://teams-api:8000
+FIFTYONE_ENV=production
+GRAPHQL_DEFAULT_LIMIT=10
+
+# FiftyOne App Configuration
+FIFTYONE_DEFAULT_APP_ADDRESS=127.0.0.1
+FIFTYONE_DEFAULT_APP_PORT=5151
+FIFTYONE_DATABASE_NAME=fiftyone
+
+# FiftyOne Teams App Configuration
```

Comment:
In `docker/legacy-auth/env.template`, we set

```
FIFTYONE_BASE_DIR=/teams/some-data-directory/
```

but this setting is absent in this file. Is that intentional?